### PR TITLE
fix: correctly handle Trezor One support

### PR DESCRIPTION
**Pull request by prusnak**
- Trezor version check works correctly only for Trezor Model T which uses version numbers 2.x.x
- this PR changes the logic so also Trezor One versions are properly handled

### CODE REVIEW

File: ethers-signers/src/trezor/app.rs [hunk 1]
```diff
@@ -73,12 +75,25 @@ impl TrezorEthereum {
...
+        let mut min_version;
 
-        // Enforce firmware version is greater than FIRMWARE_MIN_VERSION
+        match version.major {
+            1 => {
+                min_version = FIRMWARE_1_MIN_VERSION;
+            }
+            2 => {
+                min_version = FIRMWARE_2_MIN_VERSION;
+            }
```
Comment (DaniPopes) [inline ethers-signers/src/trezor/app.rs:87]:
```suggestion
        let min_version = match version.major {
            1 => FIRMWARE_1_MIN_VERSION,
            2 => FIRMWARE_2_MIN_VERSION,
```

File: ethers-signers/src/trezor/app.rs [hunk 2]
```diff
@@ -73,12 +75,25 @@ impl TrezorEthereum {
     }
 
     fn check_version(version: String) -> Result<(), TrezorError> {
-        let req = semver::VersionReq::parse(FIRMWARE_MIN_VERSION)?;
         let version = semver::Version::parse(&version)?;
+        let mut min_version;
```
Comment (DaniPopes) [inline ethers-signers/src/trezor/app.rs:79]:
```suggestion
```

**Review by DaniPopes — APPROVED** [review]
lgtm, idiomatic nit

File: ethers-signers/src/trezor/app.rs (hunk 1 above)
Comment (prusnak) [reply ethers-signers/src/trezor/app.rs]: addressed in dd108acb51141e98cfc734d7ebe2e2ed2ebbcd27

**Review by prestwich — APPROVED** [review]

**Pull request merged**